frontend/src/i18n/locales/de.yaml

```diff
@@ -8,7 +8,9 @@ home: Startseite
 navigation: Navigation
 rename: Umbenennen
 this-field-is-required: Dieses Feld ist erforderlich.
-copy-to-clipboard: In Zwischenablage kopieren
+copy-direct-link-to-clipboard: Videolink in Zwischenablage kopieren
+copy-embed-code-to-clipboard: Einbettungscode in Zwischenablage kopieren
+copy-oc-id-to-clipboard: Opencast-ID in Zwischenablage kopieren
```

Comment:
I think it would be quite nice to move these translations each into its proper place inside this YAML file. E.g. the embed code as child of `video.embed`.